test/e2e/Dockerfile

```diff
@@ -20,6 +20,16 @@ RUN apt-get update && \
 RUN mkdir -p /go/src/k8s.io
 WORKDIR /go/src/k8s.io
 
+### Kubernetes 1.33
+FROM builder As tests-1.33
+ARG KUBE_VERSION_1_33=1.33.0
+ARG KUBE_VERSION_1_33_E2E_BIN_SHA256_CHECKSUM=3c44921a69c56cda7a8c4a6f8ac081a8c5fba8742922c0707c9723ce8c4fbdb8
+
+RUN curl --fail --location https://dl.k8s.io/v${KUBE_VERSION_1_33}/kubernetes-test-linux-amd64.tar.gz | tar xvzf - --strip-components 3 kubernetes/test/bin/e2e.test kubernetes/test/bin/ginkgo
+RUN echo "${KUBE_VERSION_1_32_E2E_BIN_SHA256_CHECKSUM}" e2e.test | sha256sum --check
```

Comment:
Missed this one
Should be `KUBE_VERSION_1_33_E2E_BIN_SHA256_CHECKSUM` - that'll fix the image build failure https://github.com/digitalocean/csi-digitalocean/actions/runs/15025091601/job/42223822589?pr=635